### DISCUSSION TITLE
is there a way you can get it for education edition?

### DISCUSSION THREAD

**Discussion by Shuckle101** [HbmMods/Hbm-s-Nuclear-Tech-GIT#804] [Q&A] [answered]
Since I'm a lazy person with no money to buy Minecraft sitting with a Chromebook, is there a way to get it for education edition? if not can you make a version designed for it. although I do not expect you to.

**Comment by TheForge129** [comment]
?

**Reply by Shuckle101** [reply]
I'm not surprised someone's confused.

**Reply by Shuckle101** [reply]
All you said was ???  I have known the answer to that question. I wanted to see if people would talk if I acted stupid. I could have just said 696969 in a random discussion. But I chose this. if someone were stupid they could have said "YeP It CaN."     -Useless reply from Alex duckington.

**Comment by Shuckle101** [comment]
Me waiting for a response: Day 1.

**Comment by Antineutrino-4444** [comment, marked as answer]
...bob wont even try to port it to a newer version of java
how can you expect that he is gonna port it to a completely different platform with exactly different coding and no api whatsoever
since you put it in discussion so it is fine but the point is dont have any hope. I will be shocked when bob actually agrees to do so, if that time actually comes, call me [Doge]

**Reply by Shuckle101** [reply]
Finally, 2 People with common sense. I have known. I wanted to see if people would talk if I acted stupid. I could have just said 696969 in a random discussion. But I chose this. if someone were stupid they could have said "YeP It CaN."     -Useless reply from Alex duckington.

**Comment by Adam29Adam29** [comment]
My brother in Christ, why are you asking about a port of the mod for a C++ game (Minecraft Bedrock) on a java mod GitHub. And that's the reason why people are confused.

There is a reason why you won't get response when the question asked is ridiculous.

**Reply by Shuckle101** [reply]
Finally, 2 People with common sense. I have known. I wanted to see if people would talk if I acted stupid. I could have just said 696969 in a random discussion. But I chose this. if someone were stupid they could have said "YeP It CaN."     -Useless reply from Alex duckington.

**Comment by RagexPrince683** [comment]
This mod has nothing even remotely realistic about it, nothing about it is really even that educational.

**Reply by JameH2** [reply]
REALISM DOESNT MATTER IF IT OFFERS NOTHING TO THE GAMEPLAY

**Reply by Antineutrino-4444** [reply]
u know wat? if u want realism, find a fcking work in nuclear industry. it is going to be very realisitc :)

**Comment by RagexPrince683** [comment]
> u know wat? if u want realism, find a fcking work in nuclear industry. it is going to be very realisitc :)
Okay, do you know anyone hiring?

**Reply by LePeeperSauvage** [reply]
But see, that is part of the realism. Just like in real life you have to get a degree, and then struggle to find a job! I'm sure you'll learn many things that way.

**Comment by RagexPrince683** [comment]
> REALISM DOESNT MATTER IF IT OFFERS NOTHING TO THE GAMEPLAY
True, a fun balance of realism and gameplay is needed.
